src/node/plugins/staticDataPlugin.ts

```diff
@@ -122,44 +123,27 @@ export const staticDataPlugin: Plugin = {
     }
   },
 
-  transform(_code, id) {
-    if (server && loaderMatch.test(id) && '_importGlobMap' in server) {
-      // register this module as a glob importer
-      const { watch } = idToLoaderModulesMap[id]!
-      if (watch) {
-        ;(server as any)._importGlobMap.set(
-          id,
-          [Array.isArray(watch) ? watch : [watch]].map((globs) => {
-            const affirmed: string[] = []
-            const negated: string[] = []
-
-            for (const glob of globs) {
-              ;(glob[0] === '!' ? negated : affirmed).push(glob)
-            }
-            return { affirmed, negated }
-          })
-        )
-      }
-    }
-    return null
-  },
+  hotUpdate(ctx) {
+    if (ctx.type === 'update') return
```

Comment:
For updates, IIUC, there's no need to do anything here because the dep list is updated in `load` hook.

Reply:
No, I was wrong, it is needed. I think I tested it wrongly.